python/src/oauth_scope.py

```diff
@@ -2,7 +2,7 @@
 
 
 # Enumerate the valid OAuth scopes.
-# For details, see: https://developers.pinterest.com/docs/getting-started/scopes/
+# For details, see: https://developers.pinterest.com/docs/getting-started/authentication-and-scopes/#pinterest-scopes
```

Comment:
Change in the links length causes error on lint step of python build. 
Line 5    - 118 characters
Line  64 - 100 characters

In python/access_token.ph
Line 155 - 93 characters

Reply:
Yes. I'll merge this PR and then figure out the best way to fix the build issue.